### PR TITLE
Skip first batch

### PR DESCRIPTION
**Pull request by nlordell**
As an interim solution, we skip the first batch to avoid missing the solution submission deadline and possibly cause the second batch to fail as well.

Skipped batches do not log errors but will add a increment the "failed" metric counter as we don't want to consider them successes.

A better solution is currently being worked on.

### Test Plan

CI

### CODE REVIEW

**Comment by e00E** [comment]
Is this PR still useful if #595 is close to done?

**Comment by nlordell** [comment]
@e00E I think the value is that it unblocks other PRs.

File: driver/src/driver/stablex_driver.rs [hunk 1]
```diff
@@ -47,6 +47,23 @@ impl<'a> StableXDriver<'a> {
             return Ok(false);
         }
 
+        #[cfg(not(test))]
```
Comment (fleupold) [inline driver/src/driver/stablex_driver.rs]: nit: can we move this below `auction_processing_started`? I feel like we could still measure when we "started" the processing.

File: driver/src/metrics/stablex_metrics.rs [hunk 2]
```diff
@@ -73,6 +73,12 @@ impl StableXMetrics {
         }
     }
 
+    #[cfg(not(test))]
```
Comment (fleupold) [inline driver/src/metrics/stablex_metrics.rs]: is this macro needed to suppress a warning?

**Review by fleupold — APPROVED** [review]
Let's use this as a fix for now to give us time and do the more sophisticated logic properly.

**Review by e00E — APPROVED** [review]

**Comment by fleupold** [comment]
Looks good. Seems like a perfect of use of captain "merge-when-green"!

**Pull request merged**